src/index.ts

```diff
@@ -37,18 +39,22 @@ export interface Bundle {
 export interface ExploreOptions {
   /** Exclude "unmapped" bytes from the output */
   onlyMapped?: boolean;
-  /** Generate html */
-  html?: boolean;
-  /** Generate html ans save to specified file path  */
-  file?: string;
+  /** Output result as a string */
+  output?: {
+    format: OutputFormat;
+    /** Filename to save output to */
+    filename?: string;
+  };
   /** Disable removing prefix shared by all sources */
   noRoot?: boolean;
+  /** Replace "what" by "that" map */
```

Comment:
Replace "this" by "that" is more idiomatic.

Reply:
Done